Update attack-simulation-training-teams.md (#12737)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/attack-simulation-training-teams.md

## Teams simulation configuration

> For the tenants enrolled in the Attack simulation training for Teams Private preview, follow the below given configurations steps: 

Reviewer: ```suggestion
If your organization is enrolled in Attack simulation training for Teams Private Preview, carry out the following configuration steps: 
```
